src/csv_to_confluence.py

```diff
@@ -831,16 +852,52 @@ def upload_node_information_as_confluence_pages(self):
 
     def upload_summary_for_confluence_pages(self):
         node_ids = self._mapper.get_all_keys()
+        file_path = self.__create_error_rate_histogram_image()
+        histogram = self.__summary.create_histogram_html_element(self._confluence_parent_page, file_path)
+        self.__delete_chart_file(file_path)
+
         tbody = self.__summary.create_empty_summary_table()
         for node_id in node_ids:
-            commonname = self._mapper.get_node_value_from_mapping_dict(node_id, 'COMMON_NAME')
-            if self._confluence.does_page_exists(commonname):
-                page = self._confluence.get_page_content(commonname)
-                row = self.__summary.create_summary_table_row_from_confluence_page(commonname, page)
+            common_name = self._mapper.get_node_value_from_mapping_dict(node_id, 'COMMON_NAME')
+            if self._confluence.does_page_exists(common_name):
+                page = self._confluence.get_page_content(common_name)
+                row = self.__summary.create_summary_table_row_from_confluence_page(common_name, page)
                 tbody.find('tbody').append(row)
+
         table = self.__summary.create_summary_table_frame()
         table.append(tbody)
-        self._confluence.update_confluence_page(self._confluence_parent_page, str(table))
+        wrapped_content = self.__wrap_html_elements(histogram, table)
+        self._confluence.update_confluence_page(self._confluence_parent_page, str(wrapped_content))
+
+    def __wrap_html_elements(self, *args):
+        div = TemplatePageElementCreator().create_html_element('div')
+        for arg in args:
+            div.append(arg)
+        return div
+
+    def __create_error_rate_histogram_image(self):
+        """
+        This method collects statistical data from each node and uses them to generate a histogram
+        """
+        node_ids = self._mapper.get_all_keys()
+        valid_paths = []  # List of paths leading to newest data file of each node
+        for node_id in node_ids:
+            name_csv = self.__csv_handler.generate_node_csv_name(node_id)
+            path_csv = os.path.join(self.__working_dir, node_id, name_csv)
+            # node_dir = os.path.join(self.__working_dir, node_id, f"{node_id}_stats_{year}.csv")
+            if os.path.exists(path_csv):
+                valid_paths.append(path_csv)
+
+        save_path = os.path.join(self.__working_dir, 'src', 'resources', 'error_rates_hist.png')
```

Comment:
Your IDE path does not equal the path on the server. Remove `src` and `resources`

Reply:
I am now using the resource path that is set in the toml